app_feup/lib/controller/local_storage/AppBusStopDatabase.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:async';
+import 'package:app_feup/controller/local_storage/AppDatabase.dart';
+import 'package:app_feup/model/entities/Bus.dart';
+import 'package:app_feup/model/entities/BusStop.dart';
+import 'package:sqflite/sqflite.dart';
+
+class AppBusStopDatabase extends AppDatabase{
+
+  AppBusStopDatabase():super('busstops.db', 'CREATE TABLE busstops(stopCode TEXT, busCodes TEXT, favorited TEXT)');
+
+  Future<List<BusStop>> busStops() async {
+    // Get a reference to the database
+    final Database db = await this.getDatabase();
+
+    // Query the table for all bus stops
+    final List<Map<String, dynamic>> maps = await db.query('busstops');
+    if(maps.length == 0)
+      return new List();
+
+    final List<BusStop> stops = new List();
+    maps.forEach((stop) => stops.add(BusStop(stopCode: stop['stopCode'], buses: stop['busCodes'].split(",").map<Bus>((code) => Bus(busCode: code)).toList(), favorited: stop['favorited'] == '0' ? false : true)));
```

Comment:
`stop['favorited'] == '0' ? false : true`
is the same as
`stop['favorited'] != '0'`